assets/app/views/browse/service.html

```diff
@@ -42,72 +42,81 @@
+                              <th role="presentation"></th>
+                              <th>
+                                Service Port
+                                <!-- Show cluster IP in column header instead of table body at small screen widths to save space. -->
+                                <span class="visible-xs">({{service.spec.clusterIP}})</span>
+                              </th>
+                              <th role="presentation"></th>
+                              <th>Target Port</th>
+                            </tr>
+                          </thead>
+                          <tbody>
+                            <tr ng-repeat="portMapping in service.spec.ports | orderBy:'port'">
+                              <td>
+                                <span ng-if="portMapping.nodePort">{{portMapping.nodePort}}</span>
+                                <span ng-if="!portMapping.nodePort" class="text-muted">none</span>
+                              </td>
+                              <td role="presentation" class="text-muted">&#8594;</td>
+                              <td>{{portMapping.port}}/{{portMapping.protocol}}
+                                  <span ng-if="portMapping.name">({{portMapping.name}})</span></td>
+                              <td role="presentation" class="text-muted">&#8594;</td>
+                              <td>{{portMapping.targetPort}}</td>
+                            </tr>
+                          </tbody>
+                        </table>
+                      </div>
+                      <!-- No bottom margin so Create Route link is close to content. -->
+                      <dl class="dl-horizontal left" style="margin-bottom: 0;">
+                        <dt>Routes:</dt>
+                        <dd>
+                          <span ng-if="!routesForService.length"><em>none</em></span>
+                          <span ng-repeat="route in routesForService">
+                              <span ng-if="route | isWebRoute"><a ng-href="{{route | routeWebURL}}">{{route | routeLabel}}</a></span>
+                              <span ng-if="!(route | isWebRoute)">{{route | routeLabel}}</span>
+                              <span ng-show="!$last">, </span>
+                          </span>
+                        </dd>
+                      </dl>
+                      <div class="gutter-bottom">
+                        <a ng-href="project/{{project.metadata.name}}/createRoute?service={{service.metadata.name}}">Create route</a>
+                      </div>
+                      <annotations annotations="service.metadata.annotations"></annotations>
+                    </div>
+                  </uib-tab>
+                <uib-tab active="selectedTab.events">
+                  <uib-tab-heading>Events</uib-tab-heading>
+                  <events kind="service" ng-if="selectedTab.events"></events>
+                </uib-tab>
+              </uib-tabset>
```

Comment:
Something weird going on with the indenting here. Did you miss a closing tag? I'd expect the `uib-tab` elements to be at the same indentation level.